lib/crewai-tools/src/crewai_tools/tools/notte_function_tool/.env.example

```
@@ -0,0 +1 @@
+NOTTE_API_KEY="your_notte_api_key"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)